test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/Package/TestPackageModel.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.PackageManagement.UI.Models.Package;
+using NuGet.Packaging;
+using NuGet.Packaging.Core;
+
+namespace NuGet.PackageManagement.UI.Test.Models.Package
+{
+    internal class TestPackageModel : PackageModel
```

Comment:
Is there any alternative to creating a test model wrapper? In general, I find it's best to either use real objects or use Mocks as much as possible.

Reply:
I considered several alternatives, including using a mock as we discussed offline. However, I do not like the idea of having a mock class implementing the abstract base class that is under test. because a mock indicates that we are mocking functionality we are not testing. But I moved this test model wrapper into the tests where it's being used as a private class, so it is not used elsewhere.